integration/app_integration_test.go

```diff
@@ -1614,11 +1623,14 @@ func (p *pack) startLeafAppServers(t *testing.T, count int, extraApps []service.
 			},
 		}, extraApps...)
 
-		appServer, err := p.leafCluster.StartApp(laConf)
-		require.NoError(t, err)
-		t.Cleanup(func() { appServer.Close() })
+		configs[i] = laConf
+	}
+
+	servers, err := p.leafCluster.StartApps(configs)
+	require.NoError(t, err)
 
-		servers[i] = appServer
+	for _, appServer := range servers {
```

Comment:
Are you sure this does what you want? `appServer` will be overwritten on each iteration of the loop, right? Does the closure in cleanup capture the correct `appServer` on each iteration?

Reply:
Good catch!